components/aws/sagemaker/tests/integration_tests/component_tests/test_v2_hosting.py

```diff
@@ -64,7 +66,41 @@ def test_create_v2_endpoint(kfp_client, experiment_id, boto3_session, test_file_
             k8s_client, input_endpoint_name, "endpoints"
         )
 
-        endpoint_describe["status"]["endpointStatus"] == "InService"
+        outputs = {
```

Comment:
can you add similar check for model and endpoint config? just arn should be sufficient